dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -60,6 +60,7 @@ def get_unique_dimension_rows(self):
     def make_project_dataframe(self):
         # TODO: Can we remove NULLs at registration time?
         lk_df = self._load_data_lookup.filter("id is not NULL")
+        lk_df = self._add_time_zone(lk_df)
```

Comment:
We need to think about reordering the actions in these functions. Converting time likely has to occur immediate after filtering.